### PR TITLE
Smm

### PR DESCRIPTION
**Pull request by sanderMarcusChristensen**
Vores mega seje, helt nye Quote Generator

### CODE REVIEW

File: src/main/resources/templates/timezones/index.html
Comment (tjorne) [inline src/main/resources/templates/timezones/index.html:1]: Why these changes to this file? 
Are you testing to see if I'd notice? 😃 

Please revert changes to this file.

**Comment by tjorne** [comment]
Overall looks great. 

Would have approved the PR if not for missing sql for creating and inserting data to the quotes table.

File: src/main/resources/public/css/timezones.css
Comment (tjorne) [inline src/main/resources/public/css/timezones.css:1]: Are these two newlines really necessary?

File: src/main/resources/public/css/styles.css
Comment (tjorne) [inline src/main/resources/public/css/styles.css:1]: Are these two newlines really necessary?

**Review by tjorne — CHANGES_REQUESTED** [review]

**Review by sanderMarcusChristensen — COMMENTED** [review]
fixed now

**Review by tjorne — APPROVED** [review]

**Pull request merged**